image_enc_test.go

```diff
@@ -144,6 +157,12 @@ func TestImageEncryption(t *testing.T) {
 	if !hasEncryption(ctx, client.ContentStore(), encSpec) {
 		t.Fatal("Encrypted image does not have encrypted layers")
 	}
+	image.Name = encImageName
+	image.Target = encSpec
+	if _, err := s.Create(ctx, image); err != nil {
+		t.Fatalf("Unable to create image: %v", err)
+	}
+	ls.Delete(ctx, l, leases.SynchronousDelete)
```

Comment:
Do you have to Delete it here or can you `defer` it? Just in case of these `t.Fatal()` above there should be a Delete as well?!